vtex/loaders/legacy/productListingPage.ts

```diff
@@ -111,7 +124,7 @@ const getTermFallback = (url: URL, pageTypes: PageType[], fq: string[]) => {
    */
   const isOneTermOnly = pathList.length == 1;
 
-  if (!pageTypes.length && !fq.length && isOneTermOnly) {
+  if (!pageTypes.length && !map && !fq.length && isOneTermOnly) {
```

Comment:
Can you create named variables to better explain the conditionals here?

Reply:
Fixed